uninstall.sh

```diff
@@ -183,7 +183,7 @@ while [[ $# -gt 0 ]]; do
     --skip-cache-and-logs) opt_skip_cache_and_logs=1;;
     -f|--force) opt_force=1;;
     -q|--quiet) opt_quiet=1;;
-    -d|--dry-run) opt_dry_run=1;;
+    -n|--dry-run) opt_dry_run=1;;
```

Comment:
```suggestion
    -n|-d|--dry-run) opt_dry_run=1;;
```

For backwards compatibility. We don't want to break existing uses of this script.